ddl/backfilling_scheduler.go

```diff
@@ -308,12 +308,12 @@ func (b *ingestBackfillScheduler) setupWorkers() error {
 	}
 	b.copReqSenderPool = copReqSenderPool
 	readerCnt, writerCnt := b.expectedWorkerSize()
-	skipReg := workerpool.OptionSkipRegister[idxRecResult]{}
```

Comment:
Why remove？

Reply:
Let's make it the default skip because the feature is immature.